week-2/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity == 25; // finish this statement
```

Comment:
This variable should check if the combined total of students and mentors **is greater** than the room capacity. Please fix :)

Reply:
corrected as:
var enoughSpaceInRoom = roomMaxCapacity >= studentCount + mentorCount;